src/machine/CoffeeMachine.java

```diff
@@ -0,0 +1,111 @@
+package machine;
+
+import java.util.Scanner;
+
+class CoffeeMachine {
+
+    private int amountOfWater;
+    private int amountOfMilk;
+    private int amountOfCoffeeBeans;
+    private int amountOfDisposableCups;
+    private int money;
+    private CoffeeMachineState currentState;
+
+    CoffeeMachineState getCurrentState() {
+        return this.currentState;
+    }
+
+    CoffeeMachine(int amountOfWater, int amountOfMilk, int amountOfCoffeeBeans,
+                         int amountOfDisposableCups, int money, CoffeeMachineState currentState) {
+        this.amountOfWater = amountOfWater;
+        this.amountOfMilk = amountOfMilk;
+        this.amountOfCoffeeBeans = amountOfCoffeeBeans;
+        this.amountOfDisposableCups = amountOfDisposableCups;
+        this.money = money;
+        this.currentState = currentState;
+    }
+
+    private void displayContent() {
+        System.out.println("\nThe coffee machine has:");
+        System.out.printf("%d ml of water\n", this.amountOfWater);
+        System.out.printf("%d ml of milk\n", this.amountOfMilk);
+        System.out.printf("%d g of coffee beans\n", this.amountOfCoffeeBeans);
+        System.out.printf("%d disposable cups\n", this.amountOfDisposableCups);
+        System.out.printf("$%d of money\n", this.money);
+    }
+
+    private void prepareCoffee(int chosenOption) {
+        Coffee coffee = null;
+        switch (chosenOption) {
+            case 1 -> coffee = Coffee.ESPRESSO;
```

Comment:
We can add static method to the `Coffee` enum: `Coffee fromOption(int option)` and move this switch there.

Then here we can write: `final var coffee = Coffee.fromOption(chosenOption)`